src/components/TextInput/TextInput.tsx

```diff
@@ -59,6 +57,12 @@ export interface InputProps {
   /** Provides text for a `Label` component if `showLabel` is set to true;
    * populates an `aria-label` attribute if `showLabel` is set to false. */
   labelText: string;
+  /** The max number for a `number` TextInput type. */
+  max?: number;
```

Comment:
Not sure if this matters, but when I was testing the other day, `max` was fine as a direct prop.  It was only `min` that was showing an error.